satellite/console/mail.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package console
+
+import (
+	"net/mail"
+)
+
+const (
+	// ActivationSubject activation email subject
+	ActivationSubject = "Activate your email"
+	// InvitationSubject invitation email subject
+	InvitationSubject = ""
+	// ForgotPasswordSubject forgot password email subject
+	ForgotPasswordSubject = ""
+)
+
+// MailTemplate is implementation of satellite/mailservice.Template interface
+type MailTemplate struct {
+	to            mail.Address
+	subject       string
+	htmlPath      string
+	plainTextPath string
+}
+
+// NewMailTemplate creates new instance of MailTemplate
+func NewMailTemplate(to mail.Address, subject, prefix string) MailTemplate {
```

Comment:
I think if you remove the `MailTemplate` type, make `to` an argument to `Send` and make `subject` a template concern then you can simplify all the templates to:

```
// ProjectInvitationEmail is mailservice template for project invitation email
type ProjectInvitationEmail struct {
	UserName    string
	ProjectName string
}

func (ProjectInvitationEmail) Template() string { return "project-invitation" }
```